randovania/games/fusion/logic_database/Sector 6 NOC.txt

```diff
@@ -285,8 +288,8 @@ Extra - room_id: [6]
   > Door to Mimic Storage
       Any of the following:
           Varia Suit
-          # Wave and Plasma can stop the last 3 X
-          Plasma Beam and Wave Beam and Damage Boosts (Advanced) and Normal Damage ≥ 207
+          # Wave beam can stop the last 3 X: https://youtu.be/Ovx0rw8hV0I
```

Comment:
whatever video got uploaded here is wrong, its showcasing the beginner movement climb